starrocks-stream-load-sdk/src/main/java/com/starrocks/data/load/stream/properties/StreamLoadProperties.java

```diff
@@ -174,6 +174,17 @@ public StreamLoadTableProperties getTableProperties(String uniqueKey) {
         return tablePropertiesMap.getOrDefault(uniqueKey, defaultTableProperties);
     }
```

Comment:
we can remove `getTableProperties(String database, String table)` and `getTableProperties(String uniqueKey)` to avoid misusing